earthdata_hashdiff/compare.py

```diff
@@ -8,13 +8,15 @@
 import json
 
 from earthdata_hashdiff.generate import (
+    GEOTIFF_HASH_KEY,
     XARRAY_DECODE_DEFAULTS,
+    get_hash_from_geotiff_file,
     get_hashes_from_xarray_input,
 )
 
 
 def matches_reference_hash_file_using_xarray(
-    request_output_path: str,
+    binary_file_path: str,
```

Comment:
This is just renaming the argument to decouple the code here from the origin of using it with Harmony regression tests.